docs/whatsnew/4.0.rst

```diff
@@ -84,6 +84,16 @@ Deprecation of `sunpy.physics.solar_rotation`
 Both of these functions have been moved to `sunkit_image.coalignment` and have identical functionality and API.
 Note that `~sunpy.physics.solar_rotation.mapsequence_solar_derotate` has been renamed to `~sunkit_image.coalignment.mapsequence_coalign_by_rotation` to more accurately reflect its functionality.
 
+New dependency on ``asdf-astropy`` for ASDF support
+===================================================
+With the 4.0 release of sunpy the `asdf` plugin has been updated to support future versions of the `asdf` Python library.
+This has lead to no significant changes to how ASDF files are handled on save or load, however the plugin code is significantly simpler.
+
+When updating sunpy to 4.0 **it is important that the asdf-astropy package is installed if asdf is installed**.
+If installing sunpy with ``sunpy[all]`` or ``sunpy[asdf]`` when using pip this will happen automatically, however, if you update sunpy with ``pip install -U sunpy`` and you have previously installed asdf (implicitly or explicitly) you will need to install the ``asdf-astropy`` packag.
+**If you have installed sunpy with conda, you will need to explicitly install asdf-astropy** if you have ``asdf`` installed as the sunpy conda package does not depend on ``asdf``.
```

Comment:
I don't know what the best solution here is. Do we add asdf and asdf-astropy to the conda recipe?

Reply:
Is there an appreciable downside to adding them as dependencies for conda?  I'd prefer to avoid the potential user confusion.

Reply:
> Do we add asdf and asdf-astropy to the conda recipe?

Yes.

Reply:
We open an issue on the feedstock repo?

Reply:
Ok I shall modify this line assuming that we are going to do that to the feedstock in 4.0

Reply:
> We open an issue on the feedstock repo?

And then do this, so your local release manager doesn't forget 😆